cmd/consul-dataplane/main.go

```diff
@@ -73,6 +73,11 @@ var (
 
 	consulDNSBindAddr string
 	consulDNSPort     int
+
+	shutdownDrainListeners bool
+	shutdownGracePeriod    int
+	gracefulShutdownPath   string
+	gracefulPort           int
```

Comment:
Should this be `gracefulShutdownPort` for consistency?

Reply:
Probably not because it will eventually be used for `POST /graceful_startup` from the post-start hook too, to block application container start until the Envoy proxy is ready.